src/main/java/uk/gov/ons/ssdc/supporttool/model/messaging/dto/CollectionCase.java

```diff
@@ -0,0 +1,15 @@
+package uk.gov.ons.ssdc.supporttool.model.messaging.dto;
```

Comment:
I would probably have split the DTOs into two packages like this:
- `uk.gov.ons.ssdc.supporttool.model.dto.messaging`
- `uk.gov.ons.ssdc.supporttool.model.dto.ui`

Saves having two packages called `dto` then, which could be confusing.

Reply:
That's a lot nicer!